### PR TITLE
extra property query

### PR DESCRIPTION
**Pull request by Sild**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/property/PropertyQueryTest.java [hunk 1]
```diff
@@ -28,7 +27,9 @@
 public class PropertyQueryTest extends PropertyMethod {
     private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
-    /* #NoTicket - base tests*/
+    /**
+     * #NoTicket
+     */
     @Test
     public void testStartDateInFuture() throws Exception {
         final Property property = new Property("query-type19", "query-entity19");
```
Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/method/property/PropertyQueryTest.java]: What we test here?

Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/method/property/PropertyQueryTest.java]: Ok

Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/property/PropertyQueryTest.java]: DateFilter for correct handle the startDate field in future

File: src/test/java/com/axibase/tsd/api/method/property/PropertyQueryTest.java [hunk 2]
```diff
@@ -1392,7 +1498,120 @@ public void testLimit0() throws Exception {
...
+        insertPropertyCheck(property3);
+
+        Map<String, Object> queryObj = new HashMap<>();
+        queryObj.put("type", property.getType());
+        queryObj.put("entity", "query-entity57-b-limitentity-*");
+        queryObj.put("startDate", Util.MIN_QUERYABLE_DATE);
+        queryObj.put("endDate", Util.MAX_QUERYABLE_DATE);
+        queryObj.put("limit", 2);
+
+        assertEquals(2, calculateJsonArraySize(formatToJsonString(getProperty(queryObj))));
```
Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/method/property/PropertyQueryTest.java]: Could you please add assert message?

**Comment by rmakulov** [comment]
lgtm

**Comment by unrealwork** [comment]
ok

**Pull request merged**